### DISCUSSION TITLE
When the Memory Fixes done in the latest commit will be released?

### DISCUSSION THREAD

**Discussion by netrounds-selvakumar** [aligungr/UERANSIM#472] [Q&A]
Greetings,
Would like to know when are you planning to release the latest memory leak fixes.
Thanks.

**Comment by aligungr** [comment]
Currently I'm working on some other performance improvements. And I was planning to release after the current works.

But anyway, after your suggestion, now I'm releasing the latest memory leak fixes.

Thanks
